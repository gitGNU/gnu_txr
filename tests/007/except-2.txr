@(try)
@(try)
@(bind a "b")
@(throw e "a")
@(catch e (x))
@(output)
caught @x
@(end)
@(throw e "b")
@(finally)
@(output)
cleanup
@(end)
@(end)
@(catch e (x))
@(output)
caught @x
@(end)
@(end)
