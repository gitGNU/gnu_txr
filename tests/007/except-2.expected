caught a
cleanup
caught b
